tests/solvers/test_solvers.py

```diff
@@ -55,7 +74,33 @@ def test_multi_init(self, rb):
         for x in xs_init:
             assert rb.objective(x, None) >= solver.fun_val_opt
 
+    def test_gaussian_mixture_integration(self):
+        gm_model = GaussianMixtures(stride=1.0, size=3)
+        params_set, params_true, _ = simulate_params(1)
```

Comment:
@jizezhang same thing here with setting a seed and running multiple simulations, and also for `test_smart_initialization`

Reply:
Yes, will remember to do that!